### DISCUSSION TITLE
Possible to see the javascript uncompiled?

### DISCUSSION THREAD

**Discussion by Neophen** [fluxonui/fluxon#12] [Feature Requests]
Hey i don't like third party libs adding random js to my projects.
I've bought the licence, I wonder if i could possibly see the unminized js for the hooks/dom updates?

**Comment by andrielfn** [comment]
Hey @Neophen, I completely understand. I believe I can publish the package with non-minified JS, making it much easier to read. What do you think? Thanks for bringing this up.

**Reply by Neophen** [reply]
That would be amazing!

**Reply by andrielfn** [reply]
Done! I've just released v1.0.16 today which does not minify the JS anymore. 🙌
